src/app/SimpleSubscriptionResumptionStorage.cpp

```diff
@@ -318,44 +374,63 @@ CHIP_ERROR SimpleSubscriptionResumptionStorage::Save(SubscriptionInfo & subscrip
     ScopedNodeId subscriptionNode = ScopedNodeId(subscriptionInfo.mNodeId, subscriptionInfo.mFabricIndex);
     // Load index and update if fabric/node is new
     SubscriptionIndex subscriptionIndex;
-    LoadIndex(subscriptionIndex);
+    LoadIndex(subscriptionIndex, 1);
```

Comment:
Why 1 ?

Reply:
This code is no longer there due to the storage rework.